py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -135,6 +135,9 @@ class RemoteConnection(object):
 
     Communicates with the server using the WebDriver wire protocol:
     https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol"""
+
+    timeout = socket._GLOBAL_DEFAULT_TIMEOUT
```

Comment:
Can we have a property that we can access here with docstrings instead of people having to access a variable directly.

e.g.

```python
@socket_timeout.setter
def socket_timeout(self, value):
     # rest of code
```